docs/apm-infra-correlation.md

```diff
@@ -0,0 +1,295 @@
+# How to Enable APM-Infra Correlation 
+
+In the APM Service Dashboards, there are charts that indicate the health of
+the underlying infrastructure. The default configuration of Splunk OpenTelemetry 
+Connector automatically configures this for you, but when specifying a custom 
+configuration it is important to keep the following in mind.
+
+
+
+## Configuration to enable APM-Infra Correlation from a standalone VM (Agent)
+In order to get your infrastructure data to show up, you need to enable the 
+following in the otel-collector:
+
+- `hostmetrics` receiver
+    - cpu, memory, filesystem and network enabled
+   
+    <em>This will allow the collection of cpu, mem, disk and network metrics.</em>
+
+
+- `signalfx` exporter 
+    
+    <em>The `signalfx` exporter will aggregate the metrics from the `hostmetrics` 
+    receiver and send in metrics such as `cpu.utilization`, which are referenced in 
+    the relevant APM service charts.</em>
+
+    - `correlation` flag (enabled by default)
+
+    - `signalfx` exporter must be enabled for metrics AND traces pipeline (example below)
+
+    <em>The `correlation` flag of the `signalfx` exporter will allow the 
+    collector to make relevant API calls to correlate your spans with the 
+    infrastructure metrics, but to do this the `signalfx` exporter must be placed 
+    in the traces pipeline. This flag is enabled by default, so no specific configuration
```

Comment:
Suggested changes:

- "... , but to do this, the `signalfx` exporter ..."
- "This flag is enabled by default, so there is no specific configuration needed."
- Change "fine tune" to "adjust" or "modify".
- Change "here" to the name of the page you are linking to.

Reply:
ack to the first 3. 
Similar problems with "here", not sure how to construct an appropriate sentence with the page name.